homepage.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="header"><h1>Quiz App</h1></header>
+    <main>
+      <div class="card">
+        <section class="bookmark">
```

Comment:
I would make the whole question card a section and the bookmark should be a button.
Whenever we have interactivity, like toggling something, we use buttons.

```suggestion
      <section class="card">
        <button class="bookmark">
```